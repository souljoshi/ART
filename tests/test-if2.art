int foo(int y)
{
    int x;
    if (y < 10)
    {
        prints("if block\n");
        x = 100;
    }
    else
    {
        prints("else block\n");
        x = 50;
    }
    return x;
}

int main()
{
    printi(foo(500));
    printi(foo(3));
}