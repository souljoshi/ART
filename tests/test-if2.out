else block
50
if block
100
